source/parameters.rst

```diff
@@ -0,0 +1,204 @@
+        www = string
+        # Set to True to keep temporary directories/files after zppy completes
+        debug = boolean(default=False)
+        # The partition of the machine to run on
+        partition = string(default="")
+        # The version of E3SM Unified to use
+        e3sm_unified = option('latest', 'test', default='latest')
+        # This should be set to True if you don't want the batch jobs to be submitted
+        dry_run = boolean(default=False)
+
+        [climo]
+        # Set to True to run this section
+        active = boolean(default=True)
+        qos = string(default="regular")
+        # The number of nodes to use
+        nodes = integer(default=4)
+        # The maximum time to run
+        walltime = string(default="02:00:00")
+        # Which files to use as input
+        input_files = string(default="eam.h0")
+        # The frequency of the data. Options include "monthly", "monthly_diurnal_8xdaily"
+        frequency = string(default="monthly")
+        # The mapping file to use
+        mapping_file = string(default="")
+        # The grid to use
+        grid = string(default="")
+        # The years to run; "1:100:20" would mean process years 1-100 in 20-year increments
+        years = string_list(default=list(""))
+        exclude = boolean(default=False)
+        # The variables to process
+        vars = string(default="")
+
+        [ts]
+        # Set to True to run this section
+        active = boolean(default=True)
+        qos = string(default="regular")
+        # The number of nodes to use
+        nodes = integer(default=1)
+        # The maximum time to run
+        walltime = string(default="02:00:00")
+        # Which files to use as input
+        input_files = string(default="eam.h0")
+        # The frequency of the data. Options include "monthly", "monthly_diurnal_8xdaily"
+        frequency = string(default="monthly")
+        # The mapping file to use
+        mapping_file = string(default="")
+        # The grid to use
+        grid = string(default="")
+        area_nm = string(default="area")
+        # The variables to process
+        vars = string(default="FSNTOA,FLUT,FSNT,FLNT,FSNS,FLNS,SHFLX,QFLX,TAUX,TAUY,PRECC,PRECL,PRECSC,PRECSL,TS,TREFHT,CLDTOT,CLDHGH,CLDMED,CLDLOW,U")
+        # The years to run; "1:100:10" would mean process years 1-100 in 10-year increments
+        years = string_list(default=list(""))
+        # Days per file
+        dpf = integer(default=30)
+        tpd = integer(default=1)
+
+        [e3sm_diags]
+        # Set to True to run this section
+        active = boolean(default=True)
+        # The grid to use
+        grid = string(default="")
+        qos = string(default="regular")
+        # The number of nodes to use
+        nodes = integer(default=1)
+        # The maximum time to run
+        walltime = string(default="02:00:00")
+        # See https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
```

Comment:
For parameters with a direct correspondence to an E3SM Diags parameter, I direct users to the E3SM Diags documentation.